src/plugins/ui_feature/autofill.ts

```diff
@@ -281,7 +280,8 @@ export class AutofillPlugin extends UIPlugin {
     if (col > 0) {
       let leftPosition = { sheetId, col: col - 1, row };
       while (
-        this.getters.getEvaluatedCell(leftPosition).type !== CellValueType.empty ||
+        // this.getters.getEvaluatedCell(leftPosition).type !== CellValueType.empty ||
+        this.getters.getCorrespondingFormulaCell(leftPosition) ||
```

Comment:
do we still need the `this.getters.getCell(leftPosition)?.content` ?
Maybe we could leave `this.getters.getEvaluatedCell(leftPosition).type !== CellValueType.empty` because it's the base case and its easier to understand (and faster). WDYT?

Reply:
it does make sense yup :)